MusicApp/Utils/PaginationParams.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Utils
+{
+    public class PaginationParams
+    {
+        private const int _maxItemsPerPage = 25;
+        private int pageSize;        
+
+        public int PageNumber { get; set; } = 1;
+
+        public int PageSize
+        {
+            get => pageSize;
+            set => pageSize = value > _maxItemsPerPage ? _maxItemsPerPage : value;
+        }
+    }
+
+    public class DefaultPagination : PaginationParams
```

Comment:
`AlbumPagination` or if it's the same for the songs, keep it like this

Reply:
DONE DONE DONE